### ISSUE TITLE
git plugin scans thrice

### ISSUE THREAD

**Issue by ascheman** [kontext-e/jqassistant-plugins#15]
As described in https://groups.google.com/d/msg/jqassistant/LuYZbHaPuHw/mriLp1p7AQAJ the git plugin seems to scan a repository thrice instead of once leading to three identical but independent sub graphs. I therefore propose to scan only once.

**Comment by kontext-e** [comment]
Merged, thank you! I plan to publish v 1.1.0 of the plugins next week.

**Comment by ascheman** [comment]
Great, I am currently working on a port to JGit ... Maybe you will postpone the 1.1.0 until I am finished?

**Comment by kontext-e** [comment]
I have no time pressure, so if we can get rid of the running the command line git - would be great.
